pkgs/development/compilers/silice/default.nix

```diff
@@ -0,0 +1,86 @@
+{ stdenv, fetchFromGitHub, lib
+, cmake, pkg-config, bash, openjdk
+, libuuid, python3
+, silice, yosys, nextpnr, verilator
+, icestorm, trellis
+}:
+
+stdenv.mkDerivation rec {
+  pname = "silice";
+  version = "unstable-2022-08-05";
+
+  src = fetchFromGitHub {
+    owner = "sylefeb";
+    repo = pname;
+    rev = "e26662ac757151e5dd8c60c45291b44906b1299f";
+    sha256 = "sha256-Q1JdgDlEErutZh0OfxYy5C4aVijFKlf6Hm5Iv+1jsj4=";
+    fetchSubmodules = true;
```

Comment:
The first two submodules look pretty specific to this software but we should remove pybind11 and use ours.

https://github.com/sylefeb/Silice/blob/master/.gitmodules